From 1e3d8233d036a55e77f6b7167bc2221b7e1a4e70 Mon Sep 17 00:00:00 2001
From: Fengyu Wu <saldry@proton.me>
Date: Sat, 22 Aug 2026 15:29:50 +0800
Subject: [PATCH] ca-certificates: update to 20260816

changelog: https://salsa.debian.org/debian/ca-certificates/-/blob/master/debian/changelog

  * Update Mozilla certificate authority bundle to version 2.90
    The following certificate authorities were added (+):
    + "SECOM TLS ECC Root CA 2024"
    + "SECOM TLS RSA Root CA 2024"
    + "Telia EC TLS Root CA v3"
    + "Telia RSA TLS Root CA v3"
    The following certificate authorities were removed (-):
    - "Atos TrustedRoot 2011"
    - "Entrust Root Certification Authority"
    - "SecureSign Root CA12"
    - "ePKI Root Certification Authority"

Signed-off-by: Fengyu Wu <saldry@proton.me>
Link: https://github.com/openwrt/openwrt/pull/24839
Signed-off-by: Robert Marko <robimarko@gmail.com>
(cherry picked from commit 7484295381160529f722be7bbdaacef6a8e28b40)
Link: https://github.com/openwrt/openwrt/pull/25324
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>
---
 package/system/ca-certificates/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/system/ca-certificates/Makefile b/package/system/ca-certificates/Makefile
index e46e00e8c8..587aa572df 100644
--- a/package/system/ca-certificates/Makefile
+++ b/package/system/ca-certificates/Makefile
@@ -7,7 +7,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ca-certificates
-PKG_VERSION:=20260601
+PKG_VERSION:=20260816
 PKG_RELEASE:=1
 PKG_MAINTAINER:=
 
@@ -16,7 +16,7 @@ PKG_LICENSE_FILES:=debian/copyright
 
 PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@DEBIAN/pool/main/c/ca-certificates
-PKG_HASH:=7ab6301f7f34eef90a4d278647c260bc0762e0e14561f4649854cf4b0d4bea21
+PKG_HASH:=d939bcdd0cb058712cf4175bac76997676eb8b68fe9473765e1b40fb3d5b186a
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
-- 
2.34.1

